Jenkinsfile

```diff
@@ -97,8 +100,15 @@ node('ubuntu-zion') {
       def theStage = branch == 'master' ? 'release' : 'build'
 
       runEvaluation({ stage ->
-        nexusPolicyEvaluation(iqStage: stage, iqApplication: iqApplicationId,
-        iqScanPatterns: [[scanPattern: "container:${imageName}"]], failBuildOnNetworkError: true)}, theStage)
+        nexusPolicyEvaluation(
+          iqStage: stage,
+          iqApplication: iqApplicationId,
+          iqScanPatterns: [
+            [scanPattern: "container:${imageName}"],
+            [scanPattern: "container:${imageName}-slim"],
+          ],
```

Comment:
Question: Have you run this? 

I'm asking because I've never tried to scan two Docker images in the same evaluation. If there are policy violations, most of them will apply to both images. Do we get those violations twice, or are they aggregated? If they are not aggregated, scanning the default image should be enough.

Reply:
It's aggregated and shows 2 occurrences. Scanning both allows us to see the benefit of the slim image in cases where a default package might be in violation. Have a look at the report linked in the description of the PR. I think it looks pretty intuitive.

Reply:
@nblair If the policy violations are aggregated, I think it's better to keep them under the same IQ app. Otherwise we will get two Jira tickets for each policy violation, but the same change will fix them both. Less overhead for us.